README.md

```diff
@@ -322,6 +322,10 @@ server {
         proxy_read_timeout 24h;
         proxy_http_version 1.1;
         proxy_set_header Connection "";
+        ## Remove comments only if you use argument USE_FORWARDED_HEADERS=1 ##
+        #proxy_set_header Host $host;
```

Comment:
X-Forwarded-Host must be set to prevent a security vulnerability. There are no needs to override Host however.

Reply:
I withdraw that